### PR TITLE
Frang: tls_connection_burst does not work as expected

### PR DESCRIPTION
**Pull request by voodam**
- Fix tls_connection_burst
- Reset connection in kernel when Frang tcp_connection_* directives triggered

### CODE REVIEW

**Comment by EvgeniiMekhanik** [comment]
http://tempesta-vm.cloud.cherryservers.net:8080/job/Run%20tests/ TCP segmentation successful build

File: fw/http.c [hunk 1]
```diff
@@ -6883,6 +6884,21 @@ static TfwConnHooks http_conn_hooks = {
...
+	TfwVhost *dflt_vh = tfw_vhost_lookup_default();
+
+	if (tfw_blk_flags & TFW_BLK_ATT_REPLY && dflt_vh
+	    && dflt_vh->frang_gconf->ip_block)
+		T_WARN_NL("Directive 'block action' can't be set to\n"
+			  "    'attack reply' if 'ip_block' from 'frang_limits' group is 'on'\n"
+			  "    (this is misconfiguration, see the wiki).\n");
+	tfw_vhost_put(dflt_vh);
+
+	return 0;
```
Comment (EvgeniiMekhanik) [inline fw/http.c:6899]: We should return -1 in case when configuration is bad i think.

File: fw/http_limits.c [hunk 2]
```diff
@@ -1404,6 +1407,15 @@ frang_tls_conn_limit(FrangAcc *ra, FrangGlobCfg *conf, int hs_state)
 	switch (hs_state) {
 	case TTLS_HS_CB_FINISHED_NEW:
 		ra->history[i].tls_sess_new++;
+
+		if (conf->tls_new_conn_burst
```
Comment (EvgeniiMekhanik) [inline fw/http_limits.c:1411]: Can you explain why we move this code here?

**Comment by EvgeniiMekhanik** [comment]
I don't know why but t_frang.test_ip_block.FrangIpBlockMsg.test_on  (on your brunch in tempesta-test) is always fails now. Do you have the same problem?

File: fw/http_limits.c (hunk 2 above)
Comment (voodam) [reply fw/http_limits.c:1411]: Just not to create second counter ("i" reused in for loop, so we need "j")

**Comment by voodam** [comment]
> I don't know why but t_frang.test_ip_block.FrangIpBlockMsg.test_on (on your brunch in tempesta-test) is always fails now. Do you have the same problem?

No, on both branches am-1716-tls-conn-burst test is passed.

File: fw/filter.c [hunk 3]
```diff
@@ -83,6 +84,8 @@ tfw_filter_block_ip(const TfwAddr *addr)
 
 	T_DBG_ADDR("filter: block", addr, TFW_NO_PORT);
 
+	tfw_cli_conn_abort_all((TfwClient *)cli);
```
[the remark this answers is not in the document]
Comment (voodam) [reply fw/filter.c:87]: Fixed

File: fw/http.c (hunk 1 above)
Comment (voodam) [reply fw/http.c:6899]: Fixed, also I'll fix all tests where this misconfiguration is used.

**Review by const-t — APPROVED** [review]

**Comment by EvgeniiMekhanik** [comment]
LGTM, but i think that it is good to squash two last commits

**Comment by voodam** [comment]
> LGTM, but i think that it is good to squash two last commits

Yes, sure. "Fixup! ..." commits are used for auto-squashing: https://fle.github.io/git-tip-keep-your-branch-clean-with-fixup-and-autosquash.html

**Pull request merged**